editor/plugins/gradient_editor_plugin.h

```diff
@@ -33,25 +33,99 @@
 
 #include "editor/editor_inspector.h"
 #include "editor/editor_plugin.h"
-#include "gradient_editor.h"
+#include "editor/editor_spin_slider.h"
```

Comment:
Do not add new includes in the header if they are not necessary. Use forward declarations instead.

Reply:
Alrighty. BTW did I get the whole "public / protected / private" thing right in the header?